packages/glib.rb

```diff
@@ -15,7 +15,7 @@ class Glib < Package
   depends_on 'util_linux'
   depends_on 'six'
   
-  ENV['CFLAGS'] = "-fno-stack-protector"
+  ENV['CFLAGS'] = "-fstack-protector"
```

Comment:
Isn't the default to enable stack protection?  @satmandu: Why did you need to add this option initially?

Reply:
Either ```ENV['CFLAGS'] = "-fno-stack-protector"``` or ```ENV['CFLAGS'] = "-fstack-protector"``` solves my compilation issue!